horizons/main.py

```diff
@@ -557,13 +557,10 @@ def edit_game_map(saved_game_name):
 	@return: bool, whether loading succeeded
 	"""
 	saved_games = SavegameManager.get_saves()
-	saved_game_path = _find_map(saved_game_name, saved_games)
-	if not saved_game_path:
+	map_name = _find_map(saved_game_name, saved_games)
+	if not map_name:
 		return False
 
-	accessor = SavegameAccessor(saved_game_path, False)
```

Comment:
> I ran GUI tests and the pytest tests after the modification and nothing was explicitly broken.

I doubt that the GUI tests even cover that functionality. Especially not the command line. I mean, we just started writing tests for all the flags.

> [...] if you try to load a savegame map that is not an actual savegame, the accessor will complain that the revision is 0 and raise SavegameTooOld

That sounds like reasonable behaviour. If the flag expects an actual savegame, and the test/user doesn't supply one, that feature is supposed to throw an error.

> I agree however, that it might be better to todo_skip this test or let it fail with the temporary file generated from NamedTemporaryFile, rather than modify map loading functions.

Yes, please. Because the way it sounds is that we're changing the implementation to make a test work. :+1: for Skip.